bin/style-dictionary

```diff
@@ -57,19 +58,30 @@ program.on('command:*', function () {
   process.exit(1);
 });
 
+function displayErrors() {
```

Comment:
why do you need a function to call another function? better function name, so more readable?

Reply:
I had been passing arguments before. I agree it makes more sense to call flush directly.